events/equeue/equeue.h

```diff
@@ -183,9 +183,9 @@ int equeue_post(equeue_t *queue, void (*cb)(void *), void *event);
 // The equeue_cancel function is irq safe.
 //
 // If called while the event queue's dispatch loop is active, equeue_cancel
-// does not guarantee that the event will not not execute after it returns as
+// does not guarantee that the event will not execute after it returns as
```

Comment:
This comment needs a bit of a rewrite now that we do return something. A return of true guarantees that it will not execute.

Reply:
updated comment